tmt/base.py

```diff
@@ -702,6 +703,8 @@ class Test(Core, tmt.export.Exportable['Test']):
     duration: str = DEFAULT_TEST_DURATION_L1
     result: str = 'respect'
 
+    serial_number: int = 0
```

Comment:
I guess the docs need also an update?

https://tmt.readthedocs.io/en/stable/spec/plans.html#discover

Reply:
Fixed in 495b7bf5